scripts/zones/Apollyon/mobs/Cynoprosopi.lua

```diff
@@ -2,35 +2,34 @@
 -- Area: Apollyon NW, Floor 4
 --  Mob: Cynoprosopi
```

Comment:
Probably outside of scope for this PR, but this NM has a mechanic where its damage is increased with any remaining other mobs.  I almost got wrecked a couple months ago in ilvl gear due to that.

Reply:
That is being handled here: https://github.com/LandSandBoat/server/pull/2923/files/41567fc011161f5834570a8c5cd336b09a33aae9#diff-1142eff1ab45510b31dab1d34cd4b972321a698b6d7f5e005ad859baed0d7022R325

Reply:
for some reason that doesn't jump to the line intended

Reply:
```lua
        mobs = { "Gorynich" },
        allDeath = function(battlefield, mob)
            depowerBoss(ID.NW_APOLLYON.mob.CYNOPROSOPI)
        end,
```
its good :+1: 1:

Reply:
Thanks, I think it is because that file diff is hidden so github gets confused.